src/components/ModelTableList/CloudGroup.js

```diff
@@ -67,48 +63,57 @@ export default function CloudGroup({ activeUser, filters }) {
       modelGroup.forEach((model) => {
         const { highestStatus } = getModelStatusGroupData(model);
         cloudModels.rows.push({
+          "data-test-model-id": model?.uuid,
           columns: [
             {
+              "data-test-column": "name",
```

Comment:
What do we think of using the prefix `data-test` for test-only attributes? Typically the attribute is simply `data-test` and then the value has the additional context to it.

I think I'm ok with this as it's still understandable and grep-able.

@barrymcgee

Reply:
I must prefer having bespoke attributes as it causes a separation of concerns between functionality and testing. It makes it clear from the code what the purpose of the attribute.

It's absolutely normal to have multiple word data attributes: https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

Reply:
Sounds all good to me!